framework/src/media/audio/audio_manager.h

```diff
@@ -50,7 +50,14 @@ enum audio_manager_result_e {
 	AUDIO_MANAGER_SUCCESS = 0
 };
 
+enum audio_manager_volume_e {
```

Comment:
I think I didn't explained enough, sorry about it.
when you add something to audio_manager.h, kind of interface for another layer, you should not open everything.
If we opened below min & max value, media player & recorder doesn't need get_max_audio_gain anymore.
If you want hide min & max please move this enum to c file.
How about below?
```
<audio_manager.h>
enum audio_manager_result_e {
   ....
   AUDIO_MANAGER_VOLUME_NOT_SUPPORT = xxx // actually I suggest AUDIO_MANAGER_DEVICE_NOT_SUPPORT
}
<audio_manager.c>
enum audio_manager_volume_e {
    AUDIO_MANAGER_VOLUME_NOT_SUPPORT = AUDIO_MANAGER_DEVICE_NOT_SUPPORT,
    AUDIO_MANGER_VOLUME_MIN = 0,
    AUDIO_MANGER_VOLUME_MAX = 10
};
```
Of course you can keep current state, then remove gain functions. Choice is yours.

Reply:
Updated as suggested.